src/app/test/test.component.ts

```diff
@@ -226,12 +226,12 @@ export class TestComponent implements OnInit, AfterViewInit {
   }
 
   downloadSelected(): void {
-    const selectedReports = this.helperService.getSelectedReports(this.reports);
+    const selectedReports: Report[] = this.helperService.getSelectedReports(this.reports);
     if (selectedReports.length > 0) {
-      const queryString: string = selectedReports.reduce(
-        (totalQuery: string, selectedReport: any) => totalQuery + 'id=' + selectedReport.storageId + '&',
-        '',
-      );
+      let queryString: string = '';
+      selectedReports.forEach((report) => {
+        queryString += `id=${report.storageId}&`;
+      });
```

Comment:
Here as well, please use for...of loop instead of forEach()
```suggestion
      for (let report of selectedReportsreport) {
        queryString += `id=${report.storageId}&`;
      }
```